ember-flatpickr/src/initializers/ember-flatpickr.ts

```diff
@@ -0,0 +1,11 @@
+import flatpickr from "flatpickr";
```

Comment:
This should be using single quotes, I think.

Reply:
You're right, I've repaired the eslint config that was linting the wrong file.